.github/workflows/release.yaml

```diff
@@ -43,6 +43,8 @@ jobs:
       - name: "Build and check package - ${{ matrix.directory }} 📦"
         uses: hynek/build-and-inspect-python-package@v2
         id: baipp
+        permissions:
+          id-token: write # required for attest-build-provenance-github
```

Comment:
I think we don't need this, as it is set on L20 above.

Reply:
without this addition we get this error https://github.com/conda-incubator/conda-store/actions/runs/11843467867/job/33004483291